ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -85,6 +85,7 @@ export default class ConfirmPageContainer extends Component {
     contact: PropTypes.object,
     isOwnedAccount: PropTypes.bool,
     supportsEIP1559V2: PropTypes.bool,
+    isFailedTransaction: PropTypes.bool,
```

Comment:
The prop types for this component are organized into categories. This prop should be in the "Footer" category, but it's currently listed under "Gas Popover".